cloudify_cli/config/parser_config.py

```diff
@@ -1089,6 +1090,23 @@ def parser_config():
                 },
                 'handler': cfy.upgrade.upgrade
             },
+            'rollback': {
+                'help': 'Rollback the Manager upgrade',
```

Comment:
This may be a little misleading since it's an upgrade rollback. I do agree that there could be a better phrasing though...